integration-test/src/test/java/org/apache/iotdb/relational/it/query/recent/IoTDBTableAggregationIT.java

```diff
@@ -433,6 +433,327 @@ public void countTest() {
     tableResultSetEqualTest("select count(*) from table1", expectedHeader, retArray, DATABASE_NAME);
   }
 
+  @Test
+  public void countIfTest() {
```

Comment:
also need to add some extra unexpected exception test, like:
1. input expression is not boolean type
2. number of input expressions are not only 1